src/components/BookItem.js

```diff
@@ -1,12 +1,21 @@
+/* eslint-disable camelcase */
 import PropTypes from 'prop-types';
 import '../modules/BookItem.css';
-// eslint-disable-next-line arrow-body-style
+import { useDispatch } from 'react-redux';
+import { removedBook } from '../redux/books/bookSlice';
+
 const BookItem = ({ bookListProp }) => {
+  const dispatch = useDispatch();
+  const handleDeleteBook = (item_id) => {
+    dispatch(removedBook(item_id));
+    console.log(item_id);
+  };
+
   return (
     <li className="book-item-container">
       <p>{bookListProp.title}</p>
       <p>{bookListProp.author}</p>
-      <button type="submit">Delete</button>
+      <button onClick={() => handleDeleteBook(bookListProp.item_id)} type="submit">Delete</button>
     </li>
 
   );
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your readme description more detailed to give an overview of your whole app in a broader sense to make your readme professional 👍